lib/buildTypes/minifyJS.js

```diff
@@ -1,5 +1,4 @@
 var UglifyJS = require("uglify-js");
-var removeSourceMapUrl = require("../remove_source_map_url");
```

Comment:
Oh I see now, because you added the multiline flag to the source map regex it will remove *all* source maps from the concatenated string, is that right?